src/main/webapp/js/userMap.js

```diff
@@ -104,8 +114,98 @@ function handleData(err, userData) {
             }
         }
     });
+
+    map.addPlugin('pins', function ( layer, data, options ) {
```

Comment:
Here and in many other instances, `(` doesn't need to be followed by a whitespace, and `)` doesn't need to be preceded by a whitespace.